Makefile

```
@@ -12,7 +12,7 @@ pyevm: venv
 	. venv/bin/activate; pip install py-evm
 
 eth2spec: venv
-	git clone https://github.com/ethereum/eth2.0-specs.git; \
+	git submodule update --recursive; \
```

```suggestion
	git submodule update --init --recursive; \
```
More foolproof -- also works when this repo isn't cloned with the `--recurse-submodules` flag.